src/main/java/nextstep/subway/fares/policy/DistanceBasedFarePolicySecond.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.subway.fares.policy;
+
+import nextstep.subway.path.domain.Distance;
+
+public class DistanceBasedFarePolicySecond extends DistanceBasedFarePolicyDetails {
+    private static final int PER_DISTANCE = 5;
+    private static final int MAX_DISTANCE = 50;
+
+    public DistanceBasedFarePolicySecond() {
+        super(PER_DISTANCE);
+    }
+
+    @Override
+    public boolean fareAddable(Distance currDistance, Distance targetDistance) {
+        return currDistance.getValue() < MAX_DISTANCE
```

Comment:
```suggestion
        return currDistance.isLessThan(MAX_DISTANCE)
```

getter를 쓰면 객체 내부 값이 외부로 노출되니 이런 형태로 바꿔보면 어떨까요?

Reply:
수정해서 반영하겠습니다!